WebHostLib/templates/generate.html

```diff
@@ -150,14 +164,69 @@ <h1>Generate Game{% if race %} (Race Mode){% endif %}</h1>
+                                            <select name="location_check_points" id="location_check_points">
+                                                {% for n in range(0, 11, 1) %}
+                                                    <option {% if n == 1 %}selected="selected" {% endif %} value="{{ n }}">
+                                                        {{ n }}
+                                                    </option>
+                                                {% endfor %}
+                                            </select>
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>
+                                            <label for="minutes_to_hint">Free hints over time:</label>
+                                            <span
+                                                class="interactive"
+                                                data-tooltip="Gain enough hint points for a full hint per chosen time interval.">(?)
+                                            </span>
+                                        </td>
+                                        <td>
+                                            <select name="minutes_to_hint" id="minutes_to_hint">
+                                                <option selected="selected" value="0">
+                                                    Off
+                                                </option>
+                                                <option value="5">
+                                                    Every 5 minutes
+                                                </option>
+                                                <option value="10">
+                                                    Every 10 minutes
+                                                </option>
+                                                <option value="15">
+                                                    Every 15 minutes
+                                                </option>
+                                                <option value="30">
+                                                    Every 30 minutes
+                                                </option>
+                                                <option value="45">
+                                                    Every 45 minutes
+                                                </option>
+                                                <option value="60">
+                                                    Every 1 hour
+                                                </option>
+                                                <option value="120">
+                                                    Every 2 hours
+                                                </option>
+                                                <option value="12 hours">
```

Comment:
The value here looks broken

Reply:
Good catch. I had all these values and the displayed text swapped initially and had to fix, guess I missed one